index.html

```diff
@@ -79,9 +94,26 @@ <h2 class="professional">Professional Art Printing Data</h2>
         
       </div>
 
-      <div class="project project1">
+      <div class="project project2">
         
-        <h2 class="professional">Professional Art Printing Data</h2>
+        <h2 class="professional">Data Dashboard Healthcare</h2>
+        <p class="proj-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="lang-3">
+  
+          <ul class="buttons">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+  
+        </div>
+        <button a href="#" class="project-link-btn">See Project</a></button>
+        
+      </div>
+
+      <div class="project project3">
```

Comment:
- [ ] Kindly stick to the Figma design your design is not the same as the Figma design your overall design is very good :clap: but there are a few changes check below the comparison between your and Figma design :+1: 

**Your Design**
![Screenshot from 2023-03-30 17-58-26](https://user-images.githubusercontent.com/45226848/228846539-cddb00fd-1925-41f4-975b-d174c0b84b58.png)

**Figma Design**
![Screenshot from 2023-03-29 23-58-57](https://user-images.githubusercontent.com/45226848/228846584-1d262214-4f65-4576-9e20-470c47bb0851.png)

- [ ] If you check your cards `height` is not the same as Figma design's cards 'height' so kindly fix the `height` of all cards.
- [ ] If you check the first `card` is changed to the other `cards` you can see there is one `button` and also a background `overlay` is hidden and the `picture` is shown clearly so please add this functionality whenever someone hovers over any card the overlay will be hidden and the button will be shown :+1: check below the video :point_down: and forget about the design in the video just check when I hover over any of card it is changing the state.

[Video](https://user-images.githubusercontent.com/45226848/228850385-8c6b9d1c-92bd-43a0-a91c-7e1b922f14c9.webm)

**_Note: These changes will be implemented only in desktop version not in mobile version_**